04-swap-protocol.md

```diff
@@ -1,6 +1,8 @@
 # BOLD #4: Swap Protocol
 
-Once an order match is found on the taker order book, a swap procedure is expected to execute. The following is the protocol happy flow: 
+## Overview
+
+Once an order match is found in the taker’s order book, a swap procedure should execute. The following is the protocol happy flow: 
 
 1. Taker finds a match
 2. Taker creates the private `r_preimage` and the public `r_hash` for the atomic swap
```

Comment:
quote/base currency is wrong. Also it's not a chain. I recommend to simplify things using the BTC/DAI example from step 1:
"quote currency chain" -> "DAI payment channel"
"base currency chain" -> "BTC payment channel"